### PR TITLE
Overrides the default JSON encoder to write a ISO 8601 string

### LINKED ISSUE #40: db dumptime GMT --> EST?
Wouldn't it be better to store all time data in the db in EST?

### LINKED ISSUE #84: dump_time says "GMT" but it should say "EST"

### PR DESCRIPTION
**Pull request by natebrennand**
So this was actually a JSON marshaler problem the whole time. Now I'm much more familiar with pyscopg2 anyways....

closes #40 
closes #84 

screenshot 2015-03-03 14 17 35

-----
This does break backwards compatibility but this needs to be changed so that it's valid.

### CODE REVIEW

File: density/config/flask_config.py [hunk 1]
```diff
@@ -54,3 +55,22 @@
...
            "\n\n\tsource config/<your settings file>")
     exit(1)
+
+
+""" Creates a json encoder that returns ISO 8601 strings for datetimes
+    http://flask.pocoo.org/snippets/119/ """
+from flask.json import JSONEncoder
+
+
+class CustomJSONEncoder(JSONEncoder):
```
Comment (danrschlosser) [inline density/config/flask_config.py]: I'd name this `ISO8601JSONEncoder` for clarity

**Comment by danrschlosser** [comment]
variable names then lgtm

**Comment by natebrennand** [comment]
fixed but waiting on @thebrianzeng since this trashes backwards compatibility for anyone using it

**Comment by thebrianzeng** [comment]
lgtm, this might break chris mulligan's app though but it should be a quick fix if it does. should we keep a change log?

**Comment by natebrennand** [comment]
+1 to a changelog. but we should probably do semantic versioning as well at that point.

**Comment by natebrennand** [comment]
I added a tag to the master branch to put it at `0.1.0`.
You guys as a group should decide exactly what `1.0.0` is defined as and make a milestone.

big things necessary from my perspective:
- all CORS errors fixed
- simple load testing
- separate webapp from API into different repos
  - all small bugs on the webapp fixed too
  - webapp could continue to live at `density.adicu.com`
  - API could live at `api.density.adicu.com`?

**Pull request merged**